mobile/Yote/js/modules/product/productStyles.js

```diff
@@ -7,20 +7,35 @@ import {
   , StyleSheet
 } from 'react-native'; 
 
+// import common styles
+import common from '../../global/styles/common'; 
+
 // import colors
 import YTColors from '../../global/styles/YTColors'; 
 
-let productStyles = StyleSheet.create({
-  cardHeader: {
+const productStyles = StyleSheet.create({
+
+  // common styles
+  cell: common.cell
+  , container: common.container
+  , header: common.header
+  , header2: common.header2
+  , listSeparator: common.separator
+  , icon: common.icon
+  , input: common.input
+  , shadow: common.shadow
+  , subHeader: common.subHeader
+  , text: common.text
+  , userImg: common.userImg
```

Comment:
is this necessary?  Can't you just call `<View styles={[common.cell, productStyles.thing]}/>` or something

Reply:
I guess not but then for every component doing that you'd be importing two style sheets instead of just one